pkg/netconfig/netconfig.go

```diff
@@ -0,0 +1,339 @@
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package netconfig
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+const (
+	ipSetsMaxLen         = "65536"
+	ipTablesLoggingChain = "LOGGING"
+	ipTablesLogPrefix    = "Policy-Filter-Dropped:"
+	ipTablesLogLimit     = "10/min"
```

Comment:
I don't think it will have to be configured anytime soon. So I would keep it like that.
If we make it configurable here, we would have to make  it configurable also in shoot-networking-filter.